ingestion/functions/retrieval/scheduled_event.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
we'll use input here no? Maybe you want to have the sample code look at input to be more relevant.
up to you, no big deal if done later when we actually start using real events.

Reply:
Yep, definitely. I'll do this in another PR today.